Makefile

```diff
@@ -1,2 +1,2 @@
 notebook.min.js: notebook.js
-	uglifyjs < notebook.js > $@
+	@uglifyjs notebook.js > notebook.min.js
```

Comment:
What's the reason for changing this? (`$@` is idiomatic Make.) Is there an advantage to specifying `notebook.js` as an argument, rather than using `<`?